kong/plugins/prometheus/handler.lua

```diff
@@ -26,10 +17,7 @@ local PrometheusHandler = {
 
 function PrometheusHandler:log(_)
   local message = basic_serializer.serialize(ngx)
-  local ok, err = timer_at(0, log, message)
-  if not ok then
-    kong.log.err("failed to create timer: ", err)
-  end
+  prometheus.log(message)
```

Comment:
Isn't the zero-delay timer needed here - as a workaround for the [cosocket limitation](https://github.com/openresty/lua-nginx-module#cosockets-not-available-everywhere) in the log phase?

Reply:
We don't need cosocket for this plugin. The Prometheus plugin is primarily maintaining the shdict and doesn't have any yield operations

Reply:
Maybe add a small comment explaining why there is no timer? (to prevent future us from making copy-pasta mistakes)